### PR TITLE
Fix for APIMANAGER-3976

### PR DESCRIPTION
**Pull request by thilinicooray**
Fix for Publisher/Creator of tenants cannot download the artefacts uploaded as files for documents from API Publisher

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/overview/template.jag [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <% jagg.template("overview", function(inputs, outputs, jagg) { %>
 <% var api=outputs.api; var createPermitted=outputs.isCreatePermitted;%>
-<%var tenant= encode.forUriComponent(request.getParameter("tenant"));%>
+<%var tenant= tenantDomain;%>
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/overview/template.jag]: Formatting issue (space need before =)

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/jagg/jagg.jag [hunk 2]
```diff
@@ -693,6 +693,16 @@ return getThemePath() + path;
         return true;
     };
 
+    var getTenantDomain = function(){
+        var user = jagg.getUser(),username;
+        if(user!=null){
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/jagg/jagg.jag]: Formatting issue

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/jagg/jagg.jag [hunk 3]
```diff
@@ -693,6 +693,16 @@ return getThemePath() + path;
         return true;
     };
 
+    var getTenantDomain = function(){
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/jagg/jagg.jag]: Formatting issue

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/overview/template.jag (hunk 1 above)
Comment (thilinicooray) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/overview/template.jag]: Fixed in PR #1280

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/jagg/jagg.jag (hunk 2 above)
Comment (thilinicooray) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/jagg/jagg.jag]: Fixed in PR #1280

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/jagg/jagg.jag (hunk 3 above)
Comment (thilinicooray) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/jagg/jagg.jag]: Fixed in PR #1280

**Pull request merged**